src/Clock/Clock.php

```diff
@@ -14,19 +16,27 @@
  */
 class Clock
 {
-    /** @var ?float */
-    private static $now;
+    /** @var ?float number of seconds or null if not mocked */
```

Comment:
Nit pick: We could specify that this is the number of seconds _since the Unix Epoch (January 1 1970 00:00:00 GMT)_